src/VirtualClient/VirtualClient.Main/profiles/PERF-MONGODB.json

```diff
@@ -0,0 +1,66 @@
+{
+  "Description": "MongoDB Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "00:05:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64,win-x64,win-arm64",
+    "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu,Windows"
+  },
+  "Parameters": {},
+  "Actions": [
+    {
+      "Type": "MongoDBExecutor",
```

Comment:
mongo db is an important workload that we would like to run server/client scenario. Please add that